rnc2rng/serializer.py

```diff
@@ -199,9 +204,12 @@ def visit(self, nodes, ctx=None, indent=True):
                 self.write('</%s>' % x.name)
 
             elif x.type == DOCUMENTATION:
-                self.namespace('a')
-                fmt = '<a:documentation>%s</a:documentation>'
-                self.write(fmt % html.escape('\n'.join(x.value)))
+                if self.namespace('a') != NAMESPACES['a']:
```

Comment:
Here, I think it would be easier to follow if you just set `xmlns_attr` to `''`, and then do the `if` to give it a value as needed.